paper-icon-theme.spec

```diff
@@ -13,21 +13,25 @@
 #
 
 
+# Package Build Date
+%global checkout %(date +"%Y%m%d")
+
 # GitHub Stuff
 %global commit0 40-CHARACTER-HASH-VALUE
-
-name:		paper-icon-theme
-version:	1.0
-release:	1
-
-Summary:	Paper Icon theme
-Group:		System/GUI/Other
-License:    CC-BY-SA-4.0
-Group:      System/GUI/GNOME
-Url:        http://samuelhewitt.com/paper/icons
-Source0:    https://github.com/snwh/%{name}/archive/%{commit0}.tar.gz
-Requires:	hicolor-icon-theme, gnome-icon-theme
-BuildArch:	noarch
+%global shortcommit0 %(c=%{commit0}; echo ${c:0:7})
+
+name:         paper-icon-theme
+version:      1.0
+release:      1.%{checkout}git%{shortcommit0}
```

Comment:
If installed Copr packages are still not updated (at-least once a day), it would be better to bump up the release number with each new build and completely drop Non Numeric Release naming.